awaitility/src/main/java/com/jayway/awaitility/core/ConditionAwaiter.java

```diff
@@ -164,6 +165,20 @@ public void uncaughtException(Thread thread, Throwable throwable) {
         }
     }
 
+    private ExecutorService initExecutorService() {
+        return Executors.newSingleThreadExecutor(new ThreadFactory() {
+            public Thread newThread(Runnable r) {
+                Thread thread;
+                if (conditionSettings.hasAlias()) {
+                    thread = new Thread(Thread.currentThread().getThreadGroup(), r, "awaitility[" + conditionSettings.getAlias() + ']');
+                } else {
+                    thread = new Thread(Thread.currentThread().getThreadGroup(), r, "awaitility-thread");
```

Comment:
I was thinking about the second case - `awaitility-thread`. In case of standard Java thread naming convention the names are different.